### PR TITLE
What: move sandbox archive to separate PR

### PR DESCRIPTION
**Pull request by TheFoxAtWork**
Why:

* overloaded milestones PR #997

This change addresses the need by:

* adding in conditions for sandbox archival

### CODE REVIEW

File: process/sandbox.md [hunk 1]
```diff
@@ -91,3 +91,13 @@ Frequency of reviews can be found on the [repo's README under "Frequency"](https
...
+
+### Sandbox archival
+
+Upon entering sandbox, projects are very early and often experimental. Not all projects that enter the sandbox will leave the sandbox to move on to incubation. Some projects may enter the sandbox and not grow or see widespread adoption or interest. This could be the result of a number of factors, such as being too early in the market, the problem targeted is being addressed in other ways, etc.
+The following criteria are leveraged by the TOC to identify when a Sandbox project is no longer viable and is not expected to move to incubation. The TOC will consider any of the criteria or a combination of them for Archival evaluation.
+* Missed Annual review by more than two months
+* Stale contributions over the course of 18 months - stale contributions include non-significant contributions (minor changes like color in the UI, or grammar corrections), lack of contributions, little discussion or activity on issues, etc. 
+* Missed reasonable [milestones] - subject to the nature of the project, project may miss milestones a reasonable person would assume to have been completed considering other factors about the project.
```
Comment (craigbox) [inline process/sandbox.md:101]: I don't think the anchor will resolve.

Comment (craigbox) [reply process/sandbox.md:101]: Does this wording not turn milestones into requirements?  Who is the "reasonable person" in this situation? (Consider, if you will, a project that wishes to remain in the CNCF, or at a particular level, against the will of a section of the community to remove it.)

Is it the intention to also define the vote required to archive a project?  Will it be two-thirds, as with a project joining?

Comment (TheFoxAtWork) [reply process/sandbox.md:101]: Thats a good call out and not what is intended. Milestones are guiding points, but not requirements. If the annual review is missed, contributions have become stale, they're also not likely hitting some of those milestones either, I'm going to adjust this to make that more apparent. Thanks!

**Review by nikhita — APPROVED** [review]

File: process/sandbox.md [hunk 2]
```diff
@@ -91,3 +91,14 @@ Frequency of reviews can be found on the [repo's README under "Frequency"](https
...
+The following criteria are leveraged by the TOC to identify when a Sandbox project is no longer viable and is not expected to move to incubation. The TOC will consider any of the criteria or a combination of them for Archival evaluation.
+* Missed Annual review by more than two months
+* Stale contributions over the course of 18 months - stale contributions include non-significant contributions (minor changes like color in the UI, or grammar corrections), lack of contributions, little discussion or activity on issues, etc. 
+    * Projects with stale contributions will experience significantly delayed maturity progress (subject to the nature of the project)
+    * Projects may refer to [project milestones for Sandbox projects](https://github.com/cncf/toc/blob/master/process/project_milestones.md#sandbox) for specific activities to build maturity in their project and potentially increase contributions. 
+* Project has existed in sandbox for 4 years with little traction by adopters
```
[the remark this answers is not in the document]
Comment (craigbox) [reply process/sandbox.md:103]: 4 years is forever in Cloud Native.  If there is an "up or out" intention, I'm surprised that number is set so high.

That said, there are 5 projects in the Sandbox that are older than 4 years as of today: Telepresence, Virtual Kubelet, Network Service Mesh, OpenEBS and Strimzi.  All are, to my knowledge, active, and I'm quite surprised that none are in incubation.

Should the CNCF staff/TOC etc be prompting projects to apply?

[the remark this answers is not in the document]
Comment (rochaporto) [reply process/sandbox.md:103]: Annual reviews often had a recommendation to apply or a short list of steps to be done towards incubation.

The stale contributions for 18 months is already a good enough indication to consider the project for archival? Maybe we drop the 4 years bullet.

[the remark this answers is not in the document]
Comment (TheFoxAtWork) [reply process/sandbox.md:103]: Is it worthwhile to consider 18 months as a valid point to checkin for inactivity regardless of future review automation?

Can drop the 4 yrs , wondering if we should add any time indicator?

[the remark this answers is not in the document]
Comment (kgamanji) [reply process/sandbox.md:103]: I think it's worthwhile to drop the timeline criteria. Every project is very different in its development and process to gather more contributors.

[the remark this answers is not in the document]
Comment (craigbox) [reply process/sandbox.md]: Removing the timeline also removed _adoption_ as a criteria that the TOC can or will use to evaluate if a project is viable. Was that intentional?

[the remark this answers is not in the document]
Comment (TheFoxAtWork) [reply process/sandbox.md]: It was not, thank you @craigbox for bringing it up.

**Review by kgamanji — APPROVED** [review]
LGTM

**Comment by TheFoxAtWork** [comment]
The contents of this PR need moved to the Archiving process file instead of the process file itself.

**Comment by TheFoxAtWork** [comment]
PR is overcome by other changes in the TOC repo

**Pull request closed without merging**